src/main/java/pl/nogacz/snake/board/SoundManager.java

```diff
@@ -0,0 +1,31 @@
+//source: https://stackoverflow.com/questions/35109539/adding-sound-to-game-help-needed
+package pl.nogacz.snake.board;
+import javax.sound.sampled.*;
+import java.io.File;
+
+public class SoundManager{
+    Clip audioClip;
+    String path;
+    public SoundManager(String path) throws Exception{
+        this.path = path;
```

Comment:
Buradaki constructor içeriğini bir metot içerisine al. Aksi halde bu classın bir objesini yaratmak için bir audio dosyasına ihtiyacın olur ve testten stream açmak zorunda kalırsın. Bunun yerine Testlerde SoundManager'dan yeni bir class override et ve yeni classta yeni metodu istediğin gibi override edebilirsin. Bu yaklaşım AudioInputStream vs. kodlarını test yaparken trigger etmeden obje yaratmanı sağlar.